### DISCUSSION TITLE
How can I delete a github account for which I have lost the password?

### DISCUSSION THREAD

**Discussion by richi-rm** [community/community#56484] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have two accounts on GitHub: *richi-rm* and *lenialabs*, I want to delete lenialabs because I don't use it, but I have lost the password of the lenialabs account. How do I delete it?

**Comment by radar-blip** [comment]
Ask [support](https://support.github.com/contact/account) if they could do it for you.
